pcsx2/x86/iCore.cpp

```diff
@@ -58,7 +58,7 @@ static int s_xmmchecknext = 0;
 
 void _backupNeededXMM()
 {
-	for (int i = 0; i < iREGCNT_XMM; i++)
+	for (unsigned int i = 0; i < iREGCNT_XMM; i++)
```

Comment:
unsigned or size_t? make up your mind :P

Reply:
Forgot about this one.